coldfront/templates/common/navbar_brand.html

```diff
@@ -1,10 +1,20 @@
 {% load static %}
+{% load feature_flags %}
 <div class="bg-primary" id="brand">
   <div class="container">
     <div class="row">
       <div class="col">
-        <h1 class="float-left" style="max-width: 500px;">{{ PORTAL_NAME }} - <small>{{ PROGRAM_NAME_LONG }} Access Management System</small></h1>
-        <img src="{% static 'core/portal/imgs/brc_logo.png' %}" alt="brc_logo" width="100;" style="margin: 10px;" class="float-right">
+        <h1 class="float-left" style="max-width: 500px;">MyBRC - <small>Berkeley Research Computing Access Management System</small></h1>
```

Comment:
This should be reverted back to what it was, which dynamically displayed the correct text.